app/config/security.yml

```diff
@@ -4,7 +4,6 @@ security:
     # http://symfony.com/doc/current/book/security.html#encoding-the-user-s-password
     encoders:
         Symfony\Component\Security\Core\User\User: plaintext
-        DashboardHub\Bundle\AppBundle\Entity\User: plaintext
```

Comment:
I don't even think that it is needed because we don't store anything at all regarding the user information as this is done via OAuth.